src/glint.gleam

```diff
@@ -825,12 +834,16 @@ fn command_help_to_usage_string(help: CommandHelp, config: Config) -> String {
     Some(pretty) -> heading_style(usage_heading, pretty.usage)
   }
   <> "\n\t"
-  <> app_name
-  <> string_map(help.meta.name, string.append(" ", _))
-  <> string_map(subcommands, string.append(" ", _))
-  <> string_map(named_args, string.append(" ", _))
-  <> string_map(unnamed_args, string.append(" ", _))
-  <> string_map(flags, string.append(" ", _))
+  <> utils.wordwrap(
+    app_name
+      <> string_map(help.meta.name, string.append(" ", _))
+      <> string_map(subcommands, string.append(" ", _))
+      <> string_map(named_args, string.append(" ", _))
+      <> string_map(unnamed_args, string.append(" ", _))
+      <> string_map(flags, string.append(" ", _)),
+    max_output_width - tab_width - 2,
```

Comment:
🔧 i think it would be wise move this up in the function body and give it a name & a comment explaining the logic involved in this calculation